nsepython/rahu.py

```diff
@@ -932,4 +931,4 @@ def security_wise_archive(from_date, to_date, symbol, series="ALL"):
     base_url = "https://www.nseindia.com/api/historical/securityArchives"
     url = f"{base_url}?from={from_date}&to={to_date}&symbol={symbol.upper()}&dataType=priceVolumeDeliverable&series={series.upper()}"
     payload = nsefetch(url)
-    return pd.DataFrame(payload['data'])
\ No newline at end of file
+    return pd.DataFrame(payload['data'])
```

Comment:
btw, this line wrapping at the end is github editor's dong. I could accept it as a suggestion nevertheless